### ISSUE TITLE
nanopolish index segmentation fault

### ISSUE THREAD

**Issue by johnomics** [jts/nanopolish#222]
nanopolish index is crashing immediately:

```
$ nanopolish index
Segmentation fault (core dumped)
```

This is a git clone with v0.8.0 checked out (want a named version rather than the latest). Compilation with gcc 4.8.5 on CentOS 7.3.1611 appeared to have worked OK (despite usual flood of warnings from HDF5, so I might have missed something, but compilation of nanopolish_index.cpp reported no warnings or errors), and other commands at least show usage:

```
$ nanopolish
error: no command provided
usage: nanopolish [command] [options]
  valid commands:
    --help
    --version
    call-methylation
    eventalign
    extract
    getmodel
    help
    index
    methyltrain
    phase-reads
    scorereads
    variants
  for help on given command, type nanopolish command --help
```

```
$ nanopolish extract
extract: no paths to process

Usage: nanopolish extract [OPTIONS] <fast5|dir>...
Extract reads in fasta format

      --help                           display this help and exit
      --version                        display version
  -v, --verbose                        display verbose output
  -r, --recurse                        recurse into subdirectories
  -q, --fastq                          extract fastq (default: fasta)
  -t, --type=TYPE                      read type: template, complement, 2d, 2d-or-template, any
                                         (default: 2d-or-template)
  -b, --basecaller=NAME[:VERSION]      consider only data produced by basecaller NAME,
                                         optionally with given exact VERSION
  -o, --output=FILE                    write output to FILE (default: stdout)

Report bugs to https://github.com/jts/nanopolish/issues
```

Please could you give me a hand with this? Couldn't see anything related in the recent commits or issues, apologies if I missed something. Thanks.

**Comment by jts** [comment]
Hi John,

I've fixed this and tagged the new version as 0.8.1 for you.

Jared

**Comment by johnomics** [comment]
Excellent, thank you very much Jared! Works fine now.
